tests/manager/manager-ctor_error-003.phpt

```diff
@@ -28,4 +32,6 @@ OK: Got MongoDB\Driver\Exception\InvalidArgumentException
 Unsupported readPreference value: 'nothing'
 OK: Got MongoDB\Driver\Exception\InvalidArgumentException
 Primary read preference mode conflicts with tags
+OK: Got MongoDB\Driver\Exception\InvalidArgumentException
+Read preference tags must be an array of zero or more documents
```

Comment:
I think this should say "Primary read preference mode conflicts with tags". It doesn't matter whether they're valid or not yet.